src/main.py

```diff
@@ -104,6 +106,12 @@ def print_outdated_external_data(manifest_checker: checker.ManifestChecker):
     return len(ext_data)
 
 
+def print_errors(manifest_checker: checker.ManifestChecker):
```

Comment:
```suggestion
def print_errors(manifest_checker: checker.ManifestChecker) -> int:
```

A little weird that a function called `print_*` returns a value and doesn't print :-)

Reply:
It returns an `int` just like `print_outdated_external_data()` does.
I didn't yet figure out how to print errors in a readable and useful manner.